functions.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+function displayPDO() {
```

Comment:
poor function name, it doesnt display anything, it creates the DB connection

Reply:
rectified.